lib/widgets/scrollbar.dart

```diff
@@ -109,35 +135,9 @@ class _AppScrollbarState extends State<AppScrollbar> {
       child: MediaQuery.removePadding(
         context: context,
         removeTop: true,
-        child: NotificationListener<ScrollNotification>(
-          onNotification: (notification) {
-            if (widget.labelBuilder == null) {
-              return false;
-            }
-            final Offset? dragPosition;
-            if (notification is ScrollStartNotification) {
-              dragPosition = notification.dragDetails?.localPosition;
-            } else if (notification is ScrollUpdateNotification) {
-              dragPosition = notification.dragDetails?.localPosition;
-            } else if (notification is OverscrollNotification) {
-              dragPosition = notification.dragDetails?.localPosition;
-            } else {
-              dragPosition = null;
-            }
-            final windowSize = context.size;
-            final shouldShowScrollLabels = windowSize != null &&
-                dragPosition != null &&
-                (Directionality.of(context) == TextDirection.rtl
-                    ? (dragPosition.dx < scrollLabelDragAreaWidth && dragPosition.dx >= 0)
-                    : (dragPosition.dx > windowSize.width - scrollLabelDragAreaWidth &&
-                        dragPosition.dx <= windowSize.width));
-            if (shouldShowScrollLabels != showScrollLabels) {
-              setState(() {
-                showScrollLabels = shouldShowScrollLabels;
-              });
-            }
-            return false;
-          },
+        child: Listener(
+          onPointerDown: (event) => maybeShowLabels(event.position),
```

Comment:
```suggestion
          onPointerDown: (event) => maybeShowLabels(event.position),
          onPointerMove: (event) => maybeShowLabels(event.position),
```
Without the `onPointerMove` we are loosing the abillity to hide the label when the pointer moves out of the scroll area (see video in #91).

Reply:
Will add a test for this

Reply:
Ahh, I see it in your comment below, neat